src/gsnClient/gsnTxHelpers.ts

```diff
@@ -238,3 +240,16 @@ export const getClientId = async (): Promise<string> => {
   //convert hex to int
   return BigNumber.from(hexValue).toString();
 };
+
+export const handleGsnResponse = (txResponse: {
```

Comment:
Would be nice to just call this automatically at the end of the `relayTransaction` method. That way you don't have to follow the req/res object for quite as long, and we don't need to duplicate the call to `handleGsnResponse` in as many places.

Reply:
completely agree with this, just pushed commit with change